module/Finna/src/Finna/View/Helper/Root/Record.php

```diff
@@ -614,17 +614,13 @@ public function getCheckbox($idPrefix = '', $formAttr = false, $label = false)
      */
     public function getAllRecordImageUrls()
     {
-        $images = $this->driver->tryMethod('getAllImages', ['']);
+        $images = $this->driver->tryMethod('getAllImages', ['', false]);
         if (empty($images)) {
             return [];
         }
         $urls = [];
         foreach ($images as $image) {
-            $urls[] = $image['urls']['small'];
-            $urls[] = $image['urls']['medium'];
-            if (isset($image['urls']['large'])) {
-                $urls[] = $image['urls']['large'];
-            }
+            $urls += array_values($image['urls']);
```

Comment:
Näin ei voi tehdä. + -operaattori yhdistää taulukot avainten pohjalta, jolloin tämä toimii vain ensimmäisellä kutsulla. Sen jälkeen taulukossa on jo alkupään alkiot, eikä mitään tapahdu. `$urls = array_merge($urls, array_values($image['urls']));` voisi toimia.

Reply:
ah joo, otantani oli sen verran pieni ettei ollukkaa ku yks kuva niin eihän toi sit näkynykkää..